_includes/misc/rtc-description.md

```diff
@@ -0,0 +1,3 @@
+The Real-Time Collaboration (RTC) plugin for {{site.productname}} allows 2 or more users to collaborate on the same content in {{site.productname}} at the same time.
+
+The RTC plugin is designed to use the {{site.cloudname}} for communicating between the collaborator's editors, with all data encrypted by the user's web browser using a local encryption key. This ensures that no-one, including {{site.companyname}}, can read their content. The content in the editor can still be retrieved as HTML so {{site.productname}} can still be used for web forms, creating and editing content in Content Management Systems (CMS) and Learning Management Systems (LMS), or any application requiring a rich-text editor.
```

Comment:
`their content` reads odd? can't think of a suggestion though sorry

Reply:
resolved in https://github.com/tinymce/tinymce-docs/pull/1778/commits/9cc200b9775691b5106c568717f3069b51dbb46e